lib/committee.js

```diff
@@ -111,19 +111,22 @@ export class Committee {
 
     const retrievalResultMajority = this.#findMajority(retrievalResultProps)
     const hasRetrievalMajority = !!retrievalResultMajority
-    /** @type {CommitteeEvaluation['retrievalResult']} */
+    /** @type {CommitteeEvaluationDetails['retrievalResult']} */
```

Comment:
I think this should be called `ConsensusEvaluationDetails`, to match `consensusEvaluation`

Reply:
The type `CommitteeEvaluationDetails` describes what's stored in `committee.evaluationDetails`.

Reply:
Renamed in c9d91eb